index.html

```diff
@@ -201,7 +201,7 @@ <h5 class="intro2">LET'S CONNECT</h5>
         <a href="#"><img src="images/fingers.png" alt="fingers"></a>
     </li>
  </ul>
-
+ 
```

Comment:
- [ ] please align the get in touch of the mobile contact with the text area section like in the Figma version
- [ ] add a little bit of margin left and right to the inputs so they look like the original version

yours
![Screenshot (668)_LI](https://user-images.githubusercontent.com/65298048/206855404-a88643ae-a87d-449e-8d88-0ba522e75c6b.jpg)
figma
![Screenshot (669)](https://user-images.githubusercontent.com/65298048/206855424-1f6c7a7f-30e6-491b-a364-d52319cf8c8b.png)